package.json

```diff
@@ -64,24 +60,33 @@
     "@angular-eslint/template-parser": "19.2.1",
     "@angular/cli": "19.2.3",
     "@angular/compiler-cli": "19.2.1",
+    "@eslint/compat": "^1.2.7",
```

Comment:
Deze is niet meer nodig met de huidige config file toch?

Reply:
Zou je denken, maar hij is nog nodig om sonarjs te kunnen gebruiken:

```
...
import { fixupPluginRules } from '@eslint/compat';
...
sonarjs: fixupPluginRules(sonarjs),
...
```

Reply:
Ik heb die niet nodig bij de FF! Flow. Maar wellicht kun je die voor nu wel laten staan, zodat het bij jullie ook werkt.